lib/eco_service_web/live/eco_service_live/community_list.html.heex

```diff
@@ -1,14 +1,35 @@
+<h1 class="font-bold text-lg text-center">List of communities</h1>
 <.table id="community" rows={@communities}>
-<:col :let={community} label="Community Name"> <%= community.name%> </:col>
-<:col :let={community} label="Location Area Zone"> <%= community.location_area_zone%> </:col>
-<:col :let={community} label="Edit"> 
-    <.link phx-click="delete" phx-value-id={community.id}> <span class="text-blue-500">Delete</span> </.link>    
-</:col>
+  <:col :let={community} label="Community Name"> <%= community.name%> </:col>
+  <:col :let={community} label="Location Area Zone"> <%= community.location_area_zone%> </:col>
+  <:col :let={community} label="Delete"> 
+      <.link phx-click="delete" phx-value-id={community.id} > <span class="text-blue-500">Delete</span> </.link>    
+  </:col>
+  <:col :let={community} label="Edit"> 
+      <.link href={~p"/community/#{community.id}/edit"} > <span class="text-blue-500">Edit</span> </.link>    
+  </:col>
```

Comment:
Why do you want to have two columns in the table for the action buttons(edit and delete)? The general practice we've all followed is to have all the actions in a single column. 

Besides, why do you have a label for the column of action buttons? It's not at all a recommended practice to have a column titled delete and have many delete buttons in that column. The UX can quickly get very messy. Your UX is honestly looking very messy. Please fix that!

![CleanShot 2024-04-12 at 1  59 59@2x](https://github.com/sangeethailango/Eco-Service/assets/17314984/24df3445-e6c8-4b75-a041-5ecea1e9f80a)